app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,22 @@
+class MoviesController < ApplicationController
+  def index
+    if params[:search] == ""
+      flash[:error] = "Must provide a search query"
+      redirect_to "/discover"
+    elsif params[:search]
+      # search_term = params[:search]
+      # require "pry"; binding.pry
```

Comment:
We'll just need to remove these pry comments (lines 8, 12, and 19), and maybe the one on line 7 (I can't remember if we were leaving that for our future selves).

Reply:
Nice catch! Taking these out